plugins/modules/packaging/os/yum_versionlock.py

```diff
@@ -95,7 +95,8 @@ def get_versionlock_packages(self):
 
     def ensure_state(self, package, command):
         """ Ensure package state """
-        rc, out, err = self.module.run_command([self.yum_bin, "-q", "versionlock", command, package])
+        packages = " ".join(package)
+        rc, out, err = self.module.run_command([self.yum_bin, "-q", "versionlock", command, packages])
```

Comment:
If `package` is a list, shouldn't it better be:
```suggestion
        rc, out, err = self.module.run_command([self.yum_bin, "-q", "versionlock", command] + package)
```
?

Reply:
Could you please take a look at this change? Is this suitable?
https://github.com/ansible-collections/community.general/pull/1912/commits/c0245664b06d90f6ebb364844c10a9938701aeee

Reply:
Converting the correct form (passing a list to `run_command`) to the legacy form (passing a string to `run_command`) is something that works, but is not suitable :-) Better use my suggestion above.

Reply:
Sorry, I didn't notice the "+" and couldn't understand what you meant, corrected in https://github.com/ansible-collections/community.general/pull/1912/commits/49c8309931db2d702192e9e0be03b8f44a10e64f.

Reply:
Renaming the parameter to `packages_list` or `packages` is still a good idea :)

Reply:
Sure, Fixed in https://github.com/ansible-collections/community.general/pull/1912/commits/1fd25d95e0f0fd4935e62bca26aa6adf5e3941a2.